en/sql/schema/index_stmt.rst

```diff
@@ -43,6 +46,8 @@ For how to use indexes on the **SELECT** statement like Using SQL Hint, Descendi
     *   The session and server timezone (:ref:`timezone-parameters`) should not be changed if database contains indexes or function index on columns of type TIMESTAMP, TIMESTAMP WITH LOCAL TIME ZONE or DATETIME WITH LOCAL TIME ZONE.
     
     *   The leap second support parameter (:ref:`timezone-parameters`) should not be changed if database contains indexes or function index on columns of type TIMESTAMP or TIMESTAMP WITH LOCAL TIME ZONE. 
+    
+    *   The number of threads from PARALLEL option applies only to index loading step. The heap scan step is performed by the transaction thread (single thread). During this process, the rows are collected into batches; when a batch is full (reached 16M in size), it is dispatched to a index loading thread (or pushed in the queue), the collecting process continues. Batch size computing consists of index key size and the OID size (8 bytes).
```

Comment:
I am afraid this paragraph is too technical (for instance OID will surely confuse any reader).

Reply:
Maybe it is better to offer additional information more relevant to use, like how locking works. It first gets SCH_M lock, then demotes to IX? then it promotes in the end back to SCH_M in both cases successful or failed.

Another information worth mentioning is how unique constraint violation is checked (I think it is reported at the end of loading).

Are there other restrictions or differences to normal loading?

Reply:
We may also add an example how csql shows an index being built:
`INDEX idx1 ON usertable (field5) IN PROGRESS`

one more thing also worth addressing is `db_index` catalog table has `status` column. 
online index will be added to the catalog table once it is completed. it won't be shown during online loading. 
you may cover it as a separate patch.